frontend.py

```diff
@@ -109,7 +113,7 @@ def create_robot_sprite(robot):
     return robot_sprite
 
 
-def draw_state(state, window):
+def draw_state(state, time, window):
```

Comment:
Funkce "vykresli stav" mela predtim dva parametry: stav a okno, tedy vykreslila stav do grafickeho okenka. Nyni se mezi ty parametry vlozil "cas", kde neni jasne, o jaki cas se jedna, v dokumentacnim retezci taky o nem neni informace. Az nekde v kodu na radku 164 se najde zminka, ze se jedna o cas, kdy klient se dozvedel, ze nekdo vyhral. To bych rada videla pripsane driv.